docs/api/README.md

```diff
@@ -72,9 +100,9 @@ You can find the full [example here](../../examples/withSizeClass.js).
 
 If you only want to hide or show specific components on specific size classes, please check [Block](#block)'s `Hidden` props that conditionally hide component.
 
-### withContainerDimensions(Component) → Component
+### WithContainerDimensions
 
-Provides component with current `width` and `height` that first outer grid uses to determine size, depending on grid's `relativeTo` prop it is either viewport or grid itself.
+Provides current `width` and `height` that first outer grid uses to determine size, depending on grid's `relativeTo` prop it is either viewport or grid itself.
 
 - **width** - reference component width
 - **height** - reference component height
```

Comment:
Next line (:110), mentions the wrapper function, to make it consistent, it makes more sense to mention `WithSizeClass` component that also takes FaCC. 
Markdown relative URL for that component is: `#withsizeclass`